src/components/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -0,0 +1,37 @@
+import { FilterTabLabel, Tab, Input } from '@taskany/bricks';
+import { Text } from '@taskany/bricks/harmony';
+
+import { stripTimezone } from '../../utils/dateTime';
+
+import s from './DateRangeFilter.module.css';
+import { tr } from './DateRangeFilter.i18n';
+
+interface DateRangeFilterProps {
+    defaultFrom?: string;
+    defaultTo?: string;
+    setFrom: (date?: string) => void;
+    setTo: (date?: string) => void;
+}
+
+export const DateRangeFilter = ({ defaultFrom, defaultTo, setFrom, setTo }: DateRangeFilterProps) => {
+    return (
+        <>
+            <Tab name="date-range" label={<FilterTabLabel text={tr('Date range')} />}>
+                <div className={s.RangeContainer}>
+                    <Text>{tr('From')}:</Text>
+                    <Input
+                        type="date"
+                        defaultValue={stripTimezone(defaultFrom)}
+                        onChange={(e) => setFrom(e.target.valueAsDate?.toISOString())}
+                    />
+                    <Text>{tr('To')}:</Text>
+                    <Input
+                        type="date"
+                        defaultValue={stripTimezone(defaultTo)}
+                        onChange={(e) => setTo(e.target.valueAsDate?.toISOString())}
+                    />
+                </div>
```

Comment:
maybe it good to have warning here if user sets date "To" less then date "From"

Reply:
added